### DISCUSSION TITLE
<shift> KEY_NEXTSONG not working.

### DISCUSSION THREAD

**Discussion by LinArcX** [WayfireWM/wayfire#1551] [Q&A]
I want handle music position via pleryctl. This is what I've done so far:

```
command_forward_music = playerctl position 5+
binding_forward_music = <shift> KEY_NEXTSONG

command_backward_music = playerctl position 5-
binding_backward_music = <shift> KEY_PREVIOUSSONG
```

But it's not working. Any idea?

**Comment by Vladimir-csp** [comment]
Hi, having a similar problem: I set different actions for `KEY_NEXTSONG`, `<shift> KEY_NEXTSONG`, `<ctrl> KEY_NEXTSONG`,  but command bound to `KEY_NEXTSONG` always fires even when pressing with modifiers.
wayfire 0.7.4
